django/db/backends/base/operations.py

```diff
@@ -166,6 +166,19 @@ def deferrable_sql(self):
         """
         return ''
 
+    def fk_on_delete_sql(self, operation):
+        """
+        Return the SQL to make an ON DELETE statement during a CREATE TABLE
+        statement.
+        """
+        on_delete = ' ON DELETE %s '
+        if operation in ['CASCADE', 'NO ACTION', 'SET NULL', 'RESTRICT']:
+            return on_delete % (operation)
+        elif operation == '':
```

Comment:
The `return` makes the `else` part redundant.

```suggestion
        if operation == '':
```